### PR TITLE
commit practical01

### PR DESCRIPTION
**Pull request by vidhikapadia2799**
Car_Showroom_practical01

### CODE REVIEW

**Comment by k-ekta** [comment]
@vidhikapadia2799  Can you upload Video of this practical?

-  You can add video by clicking on _**"Attach file by dragging & dropping, selecting or pasting them."**_ . 
-  You can see this below comment section

File: practical01/src/components/GalleryCard.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="card demo" style="width: 18rem">
```
Comment (k-ekta) [inline practical01/src/components/GalleryCard.vue]: Avoid using inline css

File: practical01/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
```
Comment (k-ekta) [inline practical01/.gitignore:13]: Remove extra code here which are not needed

**Comment by vidhikapadia2799** [comment]
@k-ekta  Netlify deployed site Link :- https://cars-showrooms.netlify.app/

**Pull request merged**